snippets/objectFromPairs.md

```diff
@@ -5,7 +5,7 @@ Creates an object from the given key-value pairs.
 Use `Array.reduce()` to create and combine key-value pairs.
 
 ```js
-const objectFromPairs = arr => arr.reduce((a, v) => ((a[v[0]] = v[1]), a), {});
+const objectFromPairs = (arr) => arr.reduce((a, [key, val]) => ((a[key] = val), a), {});
```

Comment:
As far as I see on the current snippets, we don't wrap single-arguments in parenthesis do we?

Reply:
That's also one issue on Codacy

Reply:
@flxwu I though that I have to add parenthesis accordign log of Codacy but I removed them
check please again